src/SpectralAMG_MGL_Coarsener.hpp

```diff
@@ -54,8 +54,8 @@ class SpectralAMG_MGL_Coarsener : public Mixed_GL_Coarsener
               (guarantees approximation property in edge energy norm)
        @param is_hybridization_used whether to prepare the coarse space to use the
               HybridSolver
-       @param coarse_coefficient whether to construct coarse mass matrix in a way
-              so the coefficients (edge weights) can be rescaled after coarsening
+       @param coarse_components whether to store different components of coarse M
+              based on trace extensions and bubbles (for construction of coarse M)
```

Comment:
@barker29 This parameter is renamed because now both `CoefficientMBuilder` and `ElementMBuilder` have the ability to compute the rescaled coarse M (without going back to the fine grid). The difference between the two is, the former stores various components of coarse M (M_{FF}, M_{FE}, etc.), while the latter stores only the element matrices. I believe now `ElementMBuilder` is capable of doing what `CoefficientMBuilder` can do, up to the issue (about harmonic average and normal average) that we talked about some time ago.

Reply:
Great, do you recommend removing `CoefficientMBuilder` in the future?